### PR TITLE
Kaitie/ai assessment component

### PR DESCRIPTION
**Pull request by kobryan0619**
From Figma, our goal for this component was to start to work on these portions of the teacher-facing rubric.

image

[The full ticket](https://codedotorg.atlassian.net/jira/software/projects/AITT/boards/70?selectedIssue=AITT-106) describes the bounds of this work, but it is narrowly focused on the colored box that appears.  I added some structure around the other pieces, but this is not at all intended to be final.  Specifically this excludes:
-  Other content found in the `AiAssessment` component.  There will be a space to give feedback and a better image of the bot.  That is beyond the scope of this ticket. 
- Actually connecting to the AI - dummy data has been passed through for this part.

Here's what the component currently looks like:
image

## Links

[Ticket](https://codedotorg.atlassian.net/jira/software/projects/AITT/boards/70?selectedIssue=AITT-106)

## Follow up work
- [Replace placeholder text - AITT-205](https://codedotorg.atlassian.net/jira/software/projects/AITT/boards/70?selectedIssue=AITT-205)
- Write storyboard stories for this component - AITT-206.
- Get better image for the bot - AITT-207 (note that I currently took this from the figma but it looks bad here.  Working with Molly to get a better one)

### CODE REVIEW

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -68,7 +68,15 @@
   "advancedSettings": "Advanced Settings",
   "age": "Age",
   "agenda": "Agenda",
+  "aiAssessment": "AI Assessment",
+  "aiAssessmentDoesMeet": "Extensive or Convincing Evidence",
+  "aiAssessmentDoesNotMeet": "Limited or No Evidence",
   "aiAssessmentEnabled": "AI Assessment Enabled",
+  "aiAssessmentExplanation": "PLACEHOLDER TEXT FOR INFO TIP",
```
Comment (bethanyaconnor) [inline apps/i18n/common/en_us.json]: nit: don't add text that shouldn't be translated to this file

File: apps/src/templates/rubrics/AiAssessment.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  isAiAssessed,
+  studentName,
+  aiUnderstandingLevel,
+  aiConfidence,
+}) {
+  return (
+    <div>
+      <Heading6>{i18n.aiAssessment()}</Heading6>
+      <div className={style.aiAssessmentBlock}>
+        <img alt="Ai bot" src={icon} />
```
Comment (bethanyaconnor) [inline apps/src/templates/rubrics/AiAssessment.jsx]: should the alt text be translated?

File: apps/src/templates/rubrics/AiAssessmentBox.jsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    });
+  };
+
+  return (
+    <div className={boxColor()}>
+      {isAiAssessed && (
+        <div>
+          <Heading6>{studentAchievment()}</Heading6>
+          <div>
+            <EmText>{i18n.aiConfidence({aiConfidence: aiConfidence})}</EmText>
```
Comment (bethanyaconnor) [inline apps/src/templates/rubrics/AiAssessmentBox.jsx]: Can we only make this show up if `aiConfidence` is provided? I don't know if/when we'll provide that right now

**Comment by Nokondi** [comment]
I noticed there were two designs in the figma: one with the AI assessment at the top and feedback sections below the evidence levels and the other with the AI assessment and feedback sections to the right of the evidence levels. I had set up the feedback section to the right of the evidence levels, but which design are we actually planning to use?

**Review by Nokondi — APPROVED** [review]
I don't think I have anything to add here!

File: apps/src/templates/rubrics/AiAssessment.jsx (hunk 2 above)
Comment (wilkie) [reply apps/src/templates/rubrics/AiAssessment.jsx]: Yep! If you don't mind.

File: apps/i18n/common/en_us.json (hunk 1 above)
Comment (wilkie) [reply apps/i18n/common/en_us.json]: Yep. Translators see the strings in this file and its more trouble than it is worth to go hide them. Easy fix: just put a string like this as a constant directly in the module it is used.

**Review by bethanyaconnor — APPROVED** [review]
🎉

**Review by bethanyaconnor — APPROVED** [review]

**Pull request merged**